src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -121,11 +121,25 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $rootDir = $kernel->getContainer()->getParameter('kernel.root_dir');
 
         // Define Root Paths
-        $transPaths = array($rootDir.'/Resources/translations');
+        $transPaths = array();
+        if (is_dir($dir = $rootDir.'/Resources/translations')) {
+            if ($dir !== $this->defaultTransPath) {
+                $notice = sprintf('Storing translations in the "%s" directory is deprecated since Symfony 4.2. ', $dir);
```

Comment:
we are using single sentences for deprecation message: `Storing ... since Symfony 4.2, use the ...`

Reply:
fixed